### PR TITLE
v2/last schema updates 

### PR DESCRIPTION
**Pull request by calebcase**
* restructure stocks/v2NBBO and v2Trade so that the list is separate
  from the data structure allowing reuse
* fixup the bundle_spec script so it doesn't create files owned as root
  in the checkout                                                                                                      
* add a small comment so that debugging against staging is easier

### CODE REVIEW

**Comment by calebcase** [comment]
No intention of merging yet. Just staging it up for when we are ready to announce it in the docs.

File: _specs_/rest/schemas/stocks/v2NBBO.yml [hunk 1]
```diff
@@ -1,46 +1,44 @@
 type: object
 properties:
-  results:
+  # FIXME: These are part of a "Base" struct in the protobuf definitions.
```
Comment (jbonzo) [inline _specs_/rest/schemas/stocks/v2NBBO.yml:3]: what needs to be fixed?

Comment (calebcase) [reply _specs_/rest/schemas/stocks/v2NBBO.yml:3]: We need to remove the duplication.

File: _specs_/rest/schemas/common/propConfigs/conditions.yml [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
   description: |
-    Conditions of this trade.
-
-    empty: 0 (or empty array)
-    sellside: 1
-    buyside: 2
\ No newline at end of file
+    The condition code. These are the conditions of this message.  See <a
+    href="https://polygon.io/docs/get_v1_meta_conditions__ticktype__anchor"
+    alt="Condition Mappings">Condition Mappings</a> for a mapping to exchange
```
[the remark this answers is not in the document]
Comment (calebcase) [reply _specs_/rest/schemas/common/propConfigs/conditions.yml:9]: Yup, me too. This is just a copy-pasta from a block that already existed.

File: _specs_/rest/paths/stocks/v2LastNBBO.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+summary: Last NBBO for a Symbol
+description: |
+  Get the most recent NBBO tick for a given stock.
+tags:
+  - Stocks / Equities
+parameters:
+  - $ref: '../../parameters.yml#/StocksTickerPathParam'
+responses:
+  200:
```
[the remark this answers is not in the document]
Comment (calebcase) [reply _specs_/rest/paths/stocks/v2LastNBBO.yml:9]: Such a good question... I'll check

File: _specs_/rest/schemas/common/propConfigs/conditions.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jbonzo) [reply _specs_/rest/schemas/common/propConfigs/conditions.yml:9]: oh..

File: _specs_/rest/paths/stocks/v2LastNBBO.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (calebcase) [reply _specs_/rest/paths/stocks/v2LastNBBO.yml:9]: Apparently not, but I think that's a bug we need to fix.

[the remark this answers is not in the document]
Comment (jbonzo) [reply _specs_/rest/paths/stocks/v2LastNBBO.yml:9]: right that should be fixed now before we release these docs

[the remark this answers is not in the document]
Comment (calebcase) [reply _specs_/rest/paths/stocks/v2LastNBBO.yml:9]: https://github.com/polygon-io/go-app-api-last/pull/35

File: _specs_/rest/schemas/stocks/v2NBBO.yml (hunk 1 above)
Comment (calebcase) [reply _specs_/rest/schemas/stocks/v2NBBO.yml]: Removed the duplication via a v2Base.yml

File: _specs_/rest/schemas/common/propConfigs/size.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
-type: number
-format: double
-description: The size of the trade.
\ No newline at end of file
+type: integer
```
Comment (calebcase) [inline _specs_/rest/schemas/common/propConfigs/size.yml:1]: @jbonzo Just drawing special attention to this. I believe this is the right type for this field, but maybe not?

Comment (calebcase) [reply _specs_/rest/schemas/common/propConfigs/size.yml:1]: Nope, never mind. Crypto was using it and it's size is a float.

**Review by jbonzo — COMMENTED** [review]
Where is the yaml attr saying this is in dev?

**Comment by calebcase** [comment]
> Where is the yaml attr saying this is in dev?

The new flag is here:
https://github.com/polygon-io/platform-specs/pull/106/files#diff-818d16634d309084ae1c7a7b34440b9b6c40405dda579a1e7f52b70c9118300aR58-R65

And the code that acts on it is here:
https://github.com/polygon-io/ui-docs/pull/125

**Review by timetraveler328 — APPROVED** [review]

**Review by jbonzo — APPROVED** [review]

**Pull request merged**